docs/reference/content/guides/annotations.md

```diff
@@ -32,6 +32,16 @@ The [`@IndexOptions`]({{< apiref "org/mongodb/morphia/annotations/IndexOptions"
 definition.  This annotation replaces the fields found directly on the `@Index` annotation.  This annotation was added to ensure that index
 options are consistent across the various index definition approaches.
 
+#### Collation
+The [`@Collation`]({{< apiref "org/mongodb/morphia/annotations/Collation" >}}) annotation defines the options to apply to the collation of
+an index definition.  These collation definitions can significantly affect how documents are matched.  In addition to defining a collation
+as part of an index, a collation can be specified as part of a query as well.  The Options classes provide facilities for specifying a 
+specific collation to be used for any given operation.  This collation does not have to match the one defined on the index but will, of
+course, be faster if it does.  See [`CountOptions`]({{< apiref "org/mongodb/morphia/CountOptions" >}}), 
```

Comment:
Is it more that an index with a collation can support  operations that specifies a collation if the collations match? 

I recognize that I'm oversimplifying in cases of compound indexes where non-string keys are the index prefixes and operations specify a filter of ( { x: 5, y: 2, somestring: "boo" } ) or whatever.